addon/ng2/blueprints/ng2/files/e2e/tsconfig.json

```diff
@@ -4,14 +4,12 @@
     "declaration": false,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "mapRoot": "",
```

Comment:
Turns out this one needed to go away, because e2e is using tsconfig and it auto-sets the `inlineSourceMap` option which is not compatible with this one.